pkg/resources/kafka/configmap.go

```diff
@@ -63,46 +63,36 @@ func (r *Reconciler) getConfigProperties(bConfig *v1beta1.BrokerConfig, id int32
 		log.Error(err, "setting zookeeper.connect parameter in broker configuration resulted an error")
 	}
 
-	// Add SSL configuration
-	if r.KafkaCluster.Spec.ListenersConfig.SSLSecrets != nil {
-		if err := config.Set("ssl.keystore.location", serverKeystorePath+"/"+v1alpha1.TLSJKSKeyStore); err != nil {
-			log.Error(err, "setting ssl.keystore.location parameter in broker configuration resulted an error")
+	// Add Cruise Control SSL configuration
+	brokerPort, err := kafkautils.GetBrokerContainerPort(r.KafkaCluster)
+
+	if util.IsListenerPortUseSSL(r.KafkaCluster.Spec.ListenersConfig.InternalListeners, brokerPort) {
```

Comment:
Why cannot we use `IsSSLEnabledForInternalCommunication` utility function instead?

Reply:
It can, nice catch